src/calculateRentalCost.js

```diff
@@ -28,7 +28,25 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  let totalPayment = 0;
+
+  const dailyPrice = 40;
+  const maxDiscount = 50;
+  const minDiscount = 20;
+
+  if (days < 3) {
+    totalPayment = dailyPrice * days;
+  }
```

Comment:
I believe you can remove this `if` and return `dailyPrice * days` at the end of the function